src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -39,6 +46,20 @@ public void afterCompletion(
 
     // skip logging the status endpoint
     if (!url.endsWith("/status")) {
+
+      Map<String, Object> stackDriverPayload = new HashMap<>();
+      if ("post".equalsIgnoreCase(method)) {
+        String postBody =
+            new String(
+                    ((ContentCachingRequestWrapper) request).getContentAsByteArray(),
+                    StandardCharsets.UTF_8)
+                .lines()
+                .collect(Collectors.joining(""));
+        JsonNode postBodyJson = objectMapper.readTree(postBody);
+        stackDriverPayload.put("postBody", postBodyJson);
```

Comment:
Why does this need to go in a hashmap?

Reply:
I'm copying what Terra Common Lib does: https://github.com/DataBiosphere/terra-common-lib/blob/085750be68d67fed29b59302fa8b38fa944d6850/src/main/java/bio/terra/common/logging/RequestLoggingFilter.java#L92